Resources/doc/reference/configuration/match.rst

```diff
@@ -128,8 +128,11 @@ regular expressions.
 
 **type**: ``array``
 
-A list of additional HTTP status codes of the response for which to also apply
-the rule.
+By defaut, a rule will only match cacheable status codes: 200, 203, 300, 301,
```

Comment:
oh, and [travis noted](https://travis-ci.org/FriendsOfSymfony/FOSHttpCacheBundle/jobs/69496937) that you have a spelling error: s/defaut/default/